packages/examples/packages/rollup-plugin/package.json

```diff
@@ -1,59 +1,83 @@
 {
-  "name": "rollup-snap",
+  "name": "@metamask/rollup-plugin-example-snap",
   "version": "0.34.1-flask.1",
-  "private": true,
-  "description": "An example Snap built using TypeScript and Rollup",
+  "description": "MetaMask example snap demonstrating how to use the Rollup plugin to bundle a snap.",
   "repository": {
     "type": "git",
     "url": "https://github.com/MetaMask/snaps.git"
   },
   "license": "ISC",
-  "main": "src/index.js",
+  "sideEffects": false,
+  "main": "dist/bundle.js",
   "files": [
     "dist/",
+    "images/icon.svg",
     "snap.manifest.json"
   ],
   "scripts": {
     "build": "rollup --config",
-    "clean": "rimraf 'dist/*'",
-    "serve": "concurrently \"rollup --config --watch\" \"mm-snap serve\"",
-    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "build:clean": "yarn clean && yarn build",
+    "clean": "rimraf \"dist/*\"",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:changelog",
+    "lint:changelog": "yarn auto-changelog validate",
+    "lint:ci": "yarn lint",
     "lint:eslint": "eslint . --cache --ext js,ts,jsx,tsx",
-    "lint:misc": "prettier --no-error-on-unmatched-pattern --loglevel warn \"**/*.json\" \"**/*.md\" \"**/*.html\" \"!CHANGELOG.md\" --ignore-path ../../../../.gitignore",
-    "lint:ci": "yarn lint"
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:misc": "prettier --no-error-on-unmatched-pattern --loglevel warn \"**/*.json\" \"**/*.md\" \"**/*.html\" \"!CHANGELOG.md\" \"!snap.manifest.json\" --ignore-path ../../../../.gitignore",
+    "test": "yarn test:e2e",
+    "test:e2e": "jest",
+    "start": "rollup watch"
```

Comment:
Do we not need `mm-snap serve` here?

Reply:
Ideally, Rollup is able to do this without the need for `mm-snap`. I haven't looked so much into the `start` command yet. Will do that when setting up the site.